a3_procurement/doc_events/sales_invoice.py

```
@@ -1,18 +1,56 @@
 import frappe
 
+import frappe
```

Ruff (0.14.5)

Redefinition of unused `frappe` from line 1

Remove definition: `frappe`

(F811)